### PR TITLE
docs: Adding link to the documentation for self-hosting

### LINKED ISSUE #360: How to use the docker image
### Which Renovate are you using? CLI, App, or Pro
Docker image

### Which platform are you using? GitHub, GitLab, Bitbucket Azure DevOps
Whatever

### What is your question?
I want to self host my renovate bot. I found [this](https://github.com/renovatebot/renovate#self-hosting) in the documentation. But I think it's not clear.

How could I specify options or plateform? Via .env? Via config file? How to upgrade?

When I run `docker run renovate/renovate`, of course I got an error message `FATAL: You must configure a GitHub personal access token`.

I don't know what to do next.

Thank you in advance, renovate is awesome!

### PR DESCRIPTION
**Pull request by guillaumebriday**
Self hosting section is not clear in the readme IMO.

I added a link to the complete docs about it.

Closes https://github.com/renovatebot/config-help/issues/360

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ If you are not on github.com or gitlab.com, or you prefer to run your own instan
 - Run the `renovate/renovate` Docker Hub image (same content/versions as the CLI tool), run it on a schedule
 - Run the `renovate/renovate:slim` Docker Hub image if you only use package managers that don't need third party binaries (e.g. JS, Docker, Nuget, pip)
 
+[More details on the self-hosting development](https://github.com/renovatebot/renovate/blob/master/docs/development/self-hosting.md).
```
Comment (ViceIce) [inline readme.md]: Use relative link
```suggestion
[More details on the self-hosting development](docs/development/self-hosting.md).
```

**Review by viceice — APPROVED** [review]
LGTM

@rarkins What do you think?

File: readme.md (hunk 1 above)
Comment (guillaumebriday) [reply readme.md]: I did not know it was a feature, thank you @ViceIce

**Comment by rarkins** [comment]
I think it could be better to leave the link as absolute because this readme.md gets copied into various places like the npmjs.com and hub.docker.com and then the relative link will probably fail

**Comment by guillaumebriday** [comment]
Hum that's exact! I reverted it @rarkins 

Thanks

**Pull request merged**